canopy_waf_mod.F90

```diff
@@ -102,7 +102,7 @@ SUBROUTINE CANOPY_WAF( HCM, ZTOTHC, FAFRACK, FAFRACK0, UBZREF, Z0GHCM, &
             term1 = log( LAMDARS * ( ( (FLAMEH/HCM) +  1.0 - d_h ) / zo_h ) ) - &   !numerator
                 ( 1.0 + (delta*log((1.0/delta) + 1.0)) )
```

Comment:
```suggestion
                1.0 + (delta*log((1.0/delta) + 1.0))
```

![image](https://user-images.githubusercontent.com/15079414/186260118-ac1eddd0-5d50-4b5d-b298-fde6ee8576ed.png)

(unless it is wrong in the paper)

This could explain why getting negative values.

Reply:
Nice catch!  Yes, that is why.  Thanks Zach.